src/Lykke.Job.TransactionHandler/Handlers/EthereumCoreCommandHandler.cs

```diff
@@ -303,7 +303,10 @@ public class EthereumCoreCommandHandler
             string hash = queueMessage.TransactionHash;
             string cashOperationId = context.CashOperationId;
 
+            var clientAcc = await _clientAccountClient.GetByIdAsync(clientId);
+            var asset = await _assetsServiceWithCache.TryGetAssetAsync(context.AssetId);
```

Comment:
asset & clientAcc could be taken in parallel (var clientAccTask = _clientAcc... var asset = await _assets... var clientAcc = await clientAccTask)